0200    html-web-page/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Web Page</title>
+  <style>
+    table,
+    th,
+    td {
+      border: 1px solid black;
+    }
+
+    td:not(:first-child) {
+      text-align: center;
+    }
+  </style>
+</head>
+<body>
+  <!-- Header -->
+  <header>
+    <h1>Jelly of the Month Club</h1>
+
+    <h2>Make all your friends jelly of your stock of jelly!</h2>
+  </header>
+
+  <!-- Description: header page -->
+  <p>Here at JOTMC Inc., we are constantly striving to supply our customers with the highest-qualify and most unique jellies on a monthly basis.
+For the jelly lover in your life (or yourself!), this is the gift the keeps on giving all year around.
+We offer three tiers of monthly jelly joy, and each on offers their own perks.
+Here is a breakdown of each, as well as your monthly price:</p>
+
+  <!-- Table: Tier plan -->
+  <table>
+    <thead>
+      <tr>
+        <th></th>
+        <th>Basic Plan</th>
+        <th>Deluxe Plan</th>
+        <th>Jelly Giant Plan</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Three new curated jelies per month</td>
+        <td>&check;</td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Monthly jellies come with commemorative mason jars</td>
+        <td></td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Free bonus jar of preserves bi-monthly</td>
+        <td></td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Free yearly tickets to Jelly Con</td>
+        <td></td>
+        <td></td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Monthly Pricing</td>
+        <td>$30</td>
+        <td>$50</td>
+        <td>$60</td>
+      </tr>
+    </tbody>
+  </table>
+
+  <!-- Description - footpage -->
+  <p>Like what you see? Become a part of the JOTMC famliy today!</p>
+
+  <!-- Form -->
+  <form>
+    <!-- Input: text - name -->
+    <input type="text" name="name" id="name" placeholder="Your Name"/>
+
+    <!-- Input: dropdown - tier plan selection -->
+    <select name="jelly_tier_plan" id="jelly_tier_plan">
+      <option value="default">Select Your Plan:</option>
+      <option value="basic">Basic Plan</option>
+      <option value="default">Deluxe Plan</option>
+      <option value="default">Jelly Giant Plan</option>
```

Comment:
these values should be unique and descriptive to the plan the customer is signing up for.

Reply:
Updated code.